packages/design-tokens/design-token-previews/utils/previews.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import includes from "lodash/includes";
+
+import { TokenEntry } from "../types/TokenEntry";
+
+import { FontSizePreview } from "../FontSizePreview";
+import { FontWeightPreview } from "../FontWeightPreview";
+import { IconSizePreview } from "../IconSizePreview";
+
+export const createFontWeightPreview = ([
+  tokenSuffix,
+  token,
+]: TokenEntry): JSX.Element => {
+  return <FontWeightPreview fontWeight={token.value} key={tokenSuffix} />;
+};
+
+export const createFontSizePreview = ([
+  tokenSuffix,
+  token,
+]: TokenEntry): JSX.Element => {
+  return <FontSizePreview fontSize={token.value} key={tokenSuffix} />;
+};
+
+export const createIconSizePreview = ([
+  tokenSuffix,
+  token,
+]: TokenEntry): JSX.Element => {
+  return <IconSizePreview key={tokenSuffix} pixels={token.value} />;
+};
+
+export const createBorderRadiiPreview = ([
+  ,
+  token,
+]: TokenEntry): JSX.Element => {
+  return (
+    <div style={{ borderStyle: "solid", borderRadius: token.value }}>
+      &nbsp;
+    </div>
+  );
+};
+
+export const createBorderWidthPreview = ([
+  ,
+  token,
+]: TokenEntry): JSX.Element => {
+  return (
+    <div style={{ borderStyle: "solid", borderWidth: token.value }}>&nbsp;</div>
+  );
+};
+
+export const createBorderStylePreview = ([
+  ,
+  token,
+]: TokenEntry): JSX.Element => {
+  return <div style={{ borderStyle: token.value }}>&nbsp;</div>;
+};
+
+export const createColorPreview = ([, token]: TokenEntry): JSX.Element => {
+  return (
+    <div
+      style={{
+        width: "100%",
+        backgroundColor: token.value,
+      }}
+    >
+      &nbsp;
+    </div>
+  );
+};
+
+export const createSpacePreview = ([, token]: TokenEntry): JSX.Element => {
+  if (includes(token.value, "-")) {
```

Comment:
Why do we need it?

Reply:
@nyan07 Not exactly sure what you mean here. This conditional only renders the preview when the space token is NOT negative. So, the negative space tokens actually don't preview anything. I am not sure how to preview/display negative tokens.

Reply:
A token can be `-10px`,  `-20px`, etc. How people will know the difference without checking the value?
I belive they should have the same preview as a positive one, but with the minus on the value column.